### PR TITLE
add reader log info

### PR DESCRIPTION
**Pull request by 3pointer**
### What problem does this PR solve? 
add log in kafka reader 

### Check List 

Tests 

 - Unit test
 - Integration test
 - Manual test (add detailed scripts or steps below)
 - No code

Code changes

 - Has exported function/method change
 - Has exported variable/fields change
 - Has interface methods change
 - Has persistent data change

Side effects

 - Possible performance regression
 - Increased code complexity
 - Breaking backward compatibility

Related changes

 - Need to cherry-pick to the release branch
 - Need to update the documentation
 - Need to be included in the release note

### CODE REVIEW

File: tidb-binlog/driver/reader/offset.go [hunk 1]
```diff
@@ -98,11 +100,19 @@ func (ks *KafkaSeeker) seekOffsets(topic string, partitions []int32, pos int64)
 			return nil, err
 		}
 
+		log.Info("seek offsets in",
+			zap.String("topic", topic),
+			zap.Int32("partition", partition),
+			zap.Int64("start", start),
+			zap.Int64("end", end))
```
Comment (IANTHEREAL) [inline tidb-binlog/driver/reader/offset.go]: also add `zap.Int64("target ts", pos)` in log

File: tidb-binlog/driver/reader/offset.go [hunk 2]
```diff
@@ -98,11 +100,19 @@ func (ks *KafkaSeeker) seekOffsets(topic string, partitions []int32, pos int64)
...
+
 		offset, err := ks.seekOffset(topic, partition, start, end-1, pos)
 		if err != nil {
 			err = errors.Trace(err)
 			return nil, err
 		}
+
+		log.Info("seek offset success", zap.Int64("offset", offset))
```
Comment (IANTHEREAL) [inline tidb-binlog/driver/reader/offset.go]: ditto

File: tidb-binlog/driver/reader/offset.go [hunk 3]
```diff
@@ -156,18 +166,32 @@ func (ks *KafkaSeeker) seekOffset(topic string, partition int32, start int64, en
...
-	for msg := range pc.Messages() {
+	select {
+	case msg := <-pc.Messages():
 		ts, err = ks.getTSFromMSG(msg)
+
+		if err == nil {
+			log.Debug("get ts at offset success",
+				zap.String("topic", topic),
+				zap.Int32("partition", partition),
+				zap.Int64("ts", ts))
```
Comment (IANTHEREAL) [inline tidb-binlog/driver/reader/offset.go]: also add `zap.Int64("at offset", offset)`

**Review by IANTHEREAL — COMMENTED** [review]
Rest LGTM

**Comment by 3pointer** [comment]
LGTM

**Review by IANTHEREAL — APPROVED** [review]
LGTM

**Pull request merged**